.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug Report
+about: Report errors and problems
+
+---
+
+<!--
+    The Symfony Code of Conduct applies to all the activity on this repository.
+    See https://symfony.com/doc/current/contributing/code_of_conduct/index.html
+-->
+
+| Question                              | Answer
+| ------------------------------------- | ------
+| Is it a Backward Compatibility break? | yes/no
```

Comment:
Is this something that people can answer? I feel like it's not: every single bug is a BC break from a reporter pov.
To me, the BC Break label is something that can be decided by trained ppl to me.

Reply:
I don't like it either ... but was in the original Issue template, so I guessed it was OK to maintain it. Can I remove this question and only keep the "Symfony version used" question?

Reply:
You'd have my +1

Reply:
@ogizanagi I'm not against your proposal ... but I'd prefer to not do that yet. The reason is that any additional template we add complicates the decision that the issue reporter must make before opening the issue.

That decision must be as simple and clear as possible. At the moment we make the user choose between: Bug, Feature, Doc, Security, Support. That's already a lot of options. If we add "Upgrade" to report upgrade path issues, this could be too complicated.